src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -112,19 +109,12 @@ object DepositHandler extends BagValidationExtension {
     // Otherwise operators don't have the proper permissions to clean or fix the invalid zip files or bags that might stay behind
     extractAndValidatePayloadAndGetDepositReceipt(deposit, contentLength, payload, depositDir) match {
```

Comment:
Now that you're at it, perhaps you can also get rid of the pattern match as well? and perhaps even extract the `FilesPermission.changePermissionsRecursively` into one call.

Reply:
Deze snap ik niet helemaal

Reply:
Let's leave it for now. I tried a bit, but it's not really worth it. I can't get rid of calling `changePermissionsRecursively` twice.

Reply:
ok